tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -215,27 +294,28 @@ private void AddMockedResponse(HttpResponseMessage responseMessage, string apiVe
             if (expectedParams)
             {
                 queryParams.Add("api-version", apiVersion);
+                queryParams.Add("format", "text");
 
                 _httpManager.AddMockHandler(
                    new MockHttpMessageHandler
                    {
                        ExpectedMethod = HttpMethod.Get,
-                       ExpectedUrl = "http://169.254.169.254/metadata/instance/compute",
+                       ExpectedUrl = "http://169.254.169.254/metadata/instance/compute/location",
```

Comment:
should this be a constant value? Then we don't have to update everywhere if there are future changes.

Reply:
Added as a constant.